profile/build_test.go

```diff
@@ -51,22 +69,37 @@ func (b BuildTest) RemoveWorkspace() BuildTest {
 }
 
 func (b BuildTest) Build() (libcnb.BuildContext, ExpectFunc, AfterFunc) {
-	return b.context, b.expect, func() { b.RemoveWorkspace() }
+	outputPath, _ := os.MkdirTemp("/tmp", "dotprofile_out")
```

Comment:
```suggestion
	outputPath, _ := os.MkdirTemp("", "dotprofile_out")
```

I think it's better to leave this empty for portability. The temp dir can vary from OS to OS. When it's empty Go just figures that out.

> If dir is the empty string, MkdirTemp uses the default directory for temporary files, as returned by TempDir